azure-pipelines.yml

```diff
@@ -0,0 +1,60 @@
+trigger:
+- main
+
+strategy:
+ matrix:
+   Linux:
+     imageName: 'ubuntu-latest'
+   Mac:
+     imageName: 'macos-latest'
+
+pool:
+  vmImage: $(imageName)
+
+steps:
+- checkout: self
+  submodules: true
+
+- bash:  echo "##vso[task.setvariable variable=shellopts]errexit"
+  displayName: "Force exit on error (bash)"
+
+- script: |
+    sudo apt install -y libeigen3-dev \
+                        libepoxy-dev \
+                        libtheora-dev \
+                        libjpeg-dev \
+                        libpng-dev \
+                        libglu1-mesa-dev \
+                        qtbase5-dev \
+                        qtbase5-dev-tools \
+                        libqt5opengl5-dev \
+                        libfreetype6-dev \
+                        libsdl2-dev \
+                        liblua5.3-dev
```

Comment:
Could you replace `liblua5.3-dev` with `libluajit-5.1-dev`? In this case we'll be tests both 5.1 (on GNU/Linux) and 5.3 (on macOS).